index.html

```diff
@@ -5216,6 +5216,14 @@ <h3>Vocabularies</h3>
 location under W3C control.
         </p>
 
+        <p class="issue" title="How to normatively refer to vocabulary files">
```

Comment:
Since there is issue to leave this comment on, I will leave it here.

hash for vocab seems a bit odd, given we are trusting w3c to maintain both the hash, and the vocab.... an attacker with control of the origin can change both.

Reply:
Yes, I agree... which is one of the reasons why I didn't want to use hash or the vocab content in the spec in the first place. Feels like overkill.